roda-ui/roda-wui/src/main/java/org/roda/wui/client/common/slider/InfoSliderHelper.java

```diff
@@ -384,6 +383,7 @@ public static void updateInfoSliderPanel(BrowseFileBundle bundle, SliderPanel in
 
         if (StringUtils.isNotBlank(fileFormat.getMimeType())) {
           values.put(messages.viewRepresentationInfoMimetype(), createMimetypeHTML(bundle, fileFormat.getMimeType()));
+          values.put(messages.viewRepresentationInfoMimetype(), createMimetypeHTML(bundle, fileFormat.getMimeType()));
```

Comment:
I think you accidentally added this line

Reply:
Removed